.github/workflows/integration-test.yml

```diff
@@ -12,21 +12,49 @@ env:
 jobs:
   tests:
     name: Run integration tests
-    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest , windows-latest ]
+        jdk: [ 11, 17 ]
+    runs-on: ${{ matrix.os }}
+
     steps:
       - name: Checkout Branch
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.jdk }}
 
       - uses: browser-actions/setup-geckodriver@latest
       - run: geckodriver --version
 
-      - uses: browser-actions/setup-firefox@latest
-      - run: firefox --version
-          
-      - name: Download OpenSearch Security Plugin
+      - name: Set up Firefox browser for Linux
+        if: ${{ runner.os == 'Linux' }}
+        uses: browser-actions/setup-firefox@latest
+        with:
+          browser: firefox
+          version: latest
+
+      # Browser-action version does not work on Windows
+      - name: Set up Firefox browser for Windows
+        if: ${{ runner.os == 'Windows' }}
+        uses: RyanL1997/setup-browser@main
```

Comment:
I don't see any changes in this fork, lets use `browser-actions/setup-firefox` for all platforms.

Reply:
I just tested if I change it back it will be give you the error of this: 
<img width="545" alt="Screenshot 2023-01-31 at 11 17 25 AM" src="https://user-images.githubusercontent.com/109499885/215862939-0d82fd4e-1236-4039-8243-f209734957f2.png">
That's why I left that comment at L41.

Reply:
I don't see any changes in your fork, can you link the commit with the changes you needed?  I'm fine using your fork and should use the same fork for both linux/windows workflows